providers/arista/provider/provider.go

```diff
@@ -109,6 +110,10 @@ func (s *Service) Connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 	}, nil
 }
 
+func (s *Service) Disconnect(req *emptypb.Empty) error {
+	return nil
```

Comment:
maybe we should throw a 'not implemented' err instead. else the caller might assume it's been a successful disconnect where in reality we don't do anything here

Reply:
although returning nil might also be fine, else we will be getting errors every time we close something

Reply:
That was my intent. It will be called all the time, the shell closes. In most cases it will just do nothing.

But writing this, I should add a comment to make this clear.